Boot.st

```diff
@@ -83,8 +83,7 @@ devsesh onPreSaveImage.
 	maxObjects: 24576*1024
 ] ensure: [devsesh onPostSaveImage]!
 
-"Remove rogue .img and unnecessary .chg files"
-File delete: (File default: SessionManager current imagePath extension: 'img').
```

Comment:
Sorry I misread the diff when I looked at this before. I overlooked all the visual signals and thought you were adding this line to the script. That's fine then.